MekHQ/src/mekhq/gui/RepairTab.java

```diff
@@ -830,9 +814,7 @@ public void refreshTechsList() {
         // The next gets all techs who have more than 0 minutes free, and sorted by skill descending (elites at bottom)
         List<Person> techs = getCampaign().getTechs(true);
         techsModel.setData(techs);
-        if ((selected > -1) && (selected < techs.size())) {
-            techTable.setRowSelectionInterval(selected, selected);
-        }
+        filterTechs();
```

Comment:
Only change for the bugfix. Sorry it isn't in its own commit, accidentally added them both together.